Tchap/Managers/Registration/ThreePIDCredentials.swift

```diff
@@ -0,0 +1,24 @@
+/*
+ Copyright 2018 New Vector Ltd
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import Foundation
+
+/// Represent the user three PID credentials given by email verification.
```

Comment:
This structure can be used for msisdn validation too
I would update the comment with:
 "Represent the user three PID credentials defined during the verification/validation of an email or a phone number"

BTW I suggest adding in this structure the potential authentication session: a nullable `session` parameter, which is not defined in the current Tchap registration process.